news/2024/2024-08-12-the-followpoint-motion-the-ultimate-catcher.md

```diff
@@ -0,0 +1,294 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+[ExGon](https://osu.ppy.sh/users/214187) was for sure my main inspiration for a long time (and still is).
+And my current one would be [ArMa79](https://osu.ppy.sh/users/4982799) and [Dahcreeper](https://osu.ppy.sh/users/6926006). They are definitely the ones who helped me become better and let me go further by competing with me and pushing my limits.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Have you ever sustained an injury of a finger, wrist or another part of your hand during your journey? If so, how did you manage to deal with it?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+Looking back on the last 10 years, luckily not once have I had to take a long break or even quit playing because of serious injuries. But I sometimes suffer from mild pain in my fingers or wrists after a really long session. All I do is just take a small break for a day or even a few days.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+How do you feel after taking the #1 rank from [YesMyDarknesss](https://osu.ppy.sh/users/4158549), who had it for a huge while?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+It felt unreal when I had overtaken him, because when he had been dominating ctb for the last three years, I thought there was no chance for me to take it from him. But eventually I made it happen and I'm really happy for this. Never Give Up!
```

Comment:
```suggestion
It felt unreal when I had overtaken him, because when he had been dominating osu!catch for the last three years, I thought there was no chance for me to take it from him. But eventually I made it happen and I'm really happy for this. Never Give Up!
```

Reply:
Same as in https://github.com/ppy/osu-wiki/pull/11916#discussion_r1706447553